docs/brightsky.yml

```diff
@@ -499,6 +499,11 @@ components:
           type: number
           nullable: true
           example: 40
+        sunshine_10:
+          description: Sunshine duration during previous 10 minutes
+          type: number
+          nullable: true
+          example: 1080
```

Comment:
Can you adjust this `example` value? It's a bit misleading since real values will never be higher than `600`